citesphere-model/src/main/java/edu/asu/diging/citesphere/factory/impl/CitationFactory.java

```diff
@@ -141,9 +143,34 @@ public ICitation createCitation(Item item) {
 
         citation.setDateAdded(item.getData().getDateAdded());
 
-        parseExtra(data, citation);
+        if (metaData != null) {
+            parseMetaDataNote(citation, metaData);
+        } else {
+            parseExtra(data, citation);
+        }
+        
         return citation;
     }
+    
+    @Override
+    public void parseMetaDataNote(ICitation citation, Item metaData) {
+        Data data = metaData.getData();
+        if (data.getNote() == null || data.getNote().trim().isEmpty()) {
+            return;
+        }
+        
+        citation.setMetaDataItemKey(metaData.getKey());
+        citation.setMetaDataItemVersion(metaData.getVersion());
+        
+        String note = data.getNote();
+        note = note.replace("<p>", "").replace("</p>", "").trim();
```

Comment:
- merge with line 165
- I would first trim the note and then remove only the surrounding `p` tags. Otherwise we might run into the problem that someone has a `<p>` or `</p>` in one of the fields and those would be replaced as well.